packages/bpk-component-overlay/src/BpkOverlay-test.tsx

```diff
@@ -64,7 +62,7 @@ describe('BpkOverlay', () => {
 
   it('should support arbitrary props', () => {
     const { asFragment } = render(
-      <BpkOverlay testid="123">
+      <BpkOverlay>
```

Comment:
Hm I think this should still work and be tested, no? As the Overlay should accept any other props that the `div` element would accept

Reply:
Yeah it works. Just wondering what's the use of this attribute and value. Can you give some docs or reference to understand?:)

Reply:
it's an arbitrary attribute and value, it's simply to test that `BpkOverlay` accepts any other props outside those specified in the README - https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-overlay/src/BpkOverlay.js#L60

Reply:
perfect!

Reply:
Great! So it should be ready for review again:)